codegen/templates/api.mustache

```diff
@@ -90,9 +90,9 @@ export class {{classname}} {
         let contentTypes = [{{#consumes}}'{{& mediaType}}'{{#hasMore}}, {{/hasMore}}{{/consumes}}];
         let accepts = [{{#produces}}'{{& mediaType}}'{{#hasMore}}, {{/hasMore}}{{/produces}}];
         let returnType = 'Blob';
-
+        {{#vendorExtensions.x-codegen-isArrayResponse}}
```

Comment:
out of curiosity, what does this syntax do?

Reply:
So `x-codegen-isArrayResponse` is a customization we added such that the generator knows what this value means when it reads this template file. This is defined [here in our JavascriptClientCodegenGenerator.java](https://github.com/Asana/node-asana/blob/master/codegen/swagger/src/main/java/com/asana/codegen/JavascriptClientCodegenGenerator.java#L117-L119)

The `{{#<SOME_VALUE>}}{{/<SOME_VALUE>}}` is a [handlebars](https://handlebarsjs.com/) / [mustache](https://mustache.github.io/) templating language syntax. You can think of it like a div in HTML.

So what we are doing is saying, if the value of `x-codegen-isArrayResponse` is `true` write this in the actual code file that gets generated.

```
{{#vendorExtensions.x-codegen-isArrayResponse}}
CODE HERE
{{/vendorExtensions.x-codegen-isArrayResponse}}
```